code-challenges.test.js

```diff
@@ -1,5 +1,7 @@
 // ASSESSMENT 2: Coding Practical Questions with Jest
 
+const { array } = require("yargs")
```

Comment:
This will sometimes get added to your code when working with jest. If tests are failing and you have no idea why, always good to check if this is here and delete / comment it out.